compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -219,6 +219,20 @@ getVectorPreProcStrategy(linalg::LinalgOp linalgOp) {
     return VectorPreProcStrategy::None;
   }
 
+  // Walk the linalgOp code, if there is any instruction that could result in
+  // undefined behavior in mask strategy, fall back to using peel strategy.
```

Comment:
nit: let's be consistent with other comments and variable naming.

```suggestion
  // undefined behavior in masking strategy, fall back to using peeling strategy.
```

Reply:
done.